bin/_config.sh

```diff
@@ -60,25 +58,24 @@ function _docker_run() {
         i=$(( i+1 ))
     done
 
-    # Check that minishift is running
-    if ${OC_LOGIN} && ! minishift status | grep Running > /dev/null ; then
-        echo "Error: minishift is not running. Please start it first."
-        exit 1
-    fi
-
     # Check that the current user is already logged in minishift
-    if ${OC_LOGIN} && ! oc whoami &> /dev/null; then
-        echo "Error: you need to login to minishift first."
+    if ! oc whoami &> /dev/null; then
+        echo "Error: you need to login to openshift first."
         exit 1
     fi
 
-    docker run --rm -it \
+    OPENSHIFT_URL=$(oc version|grep "Server "|awk -F'Server ' '{print $2}')
+    OPENSHIFT_DOMAIN=$(echo "${OPENSHIFT_URL}" |sed 's#https://\(.*\):8443#\1#')
+
+    [[ "$USE_TTY" == "false" ]] && TTY_OPTION="" || TTY_OPTION="t"
+
+    docker run --rm -i${TTY_OPTION} \
         -u "$(id -u)" \
         --env-file "$env_file" \
         --env OC_LOGIN \
         --env K8S_AUTH_API_KEY="$(oc whoami -t)" \
-        --env K8S_AUTH_HOST="https://$(minishift ip):8443" \
-        --env MINISHIFT_IP="$(minishift ip)" \
+        --env K8S_AUTH_HOST="${OPENSHIFT_URL}" \
+        --env OPENSHIFT_DOMAIN="${OPENSHIFT_DOMAIN}" \
```

Comment:
If it's the same variable name, you can pass it to docker run with: `-e OPENSHIFT_DOMAIN`